src/main/scala/gitbucket/core/plugin/Plugin.scala

```diff
@@ -190,6 +190,26 @@ abstract class Plugin {
   def suggestionProviders(registry: PluginRegistry, context: ServletContext, settings: SystemSettings): Seq[SuggestionProvider] = Nil
 
   /**
+    * Override and set true if this plugin is theme plugin.
+    */
+  val isThemePlugin: Boolean = false
```

Comment:
Why this property is needed? It seems good to register overridden values of `themeCssFiles` and `themeSkinNames` to `PluginRegistry` simply.